emails/cc-service-reactivated.md

```diff
@@ -0,0 +1,11 @@
+# Payment successful: Your account accessing The Graph is active
+
+Hello,
+
+Thank you for using The Graph. Your payment of ${{ paymentReceivedAmountUSD }} was processed for your past month's usage with the credit card we have on file. You may review your invoice under the Billing tab in <a href="https://thegraph.com/studio/billing/">Subgraph Studio</a>.
```

Comment:
Do we need two separate templates with different wording for reactivation and shutoff?

Reply:
I assume we would do so in the notification router itself? Since we should have the data there based off what processed their payment?

Reply:
But yes, there is a separate template for crypto and credit card reactivation

Reply:
I wondered if we should use Liquid `if / else` and minimize the number of notification types, but I ended up feeling that what you did here is the simplest and most maintainable, especially at the intersection of our team & Brian & BK.